wss.js

```diff
@@ -50,9 +50,6 @@ async function getStatusUpdateData(hospitalId) {
     include: { all: true },
     where: {
       HospitalId: hospitalId,
-      currentDeliveryStatus: {
-        [Op.lt]: 'RETURNED TO SERVICE',
```

Comment:
Instead of removing this condition entirely, we should filter out explicitly the three states we know aren't displayed to the hospital- RETURNED TO SERVICE, CANCEL_ACKNOWLEDGED, REDIRECT_ACKNOWLEDGED. I think it should work if you change the op to `Op.notIn` and then put the states into an array. 

Otherwise, this query will continue to return ALL ringdowns, even completed ones, back to the beginning of time, resulting in bigger and bigger and bigger response payloads...